formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/internal/Helpers.kt

```diff
@@ -12,10 +12,29 @@ import kotlinx.serialization.modules.*
 import kotlinx.serialization.protobuf.*
 
 internal typealias ProtoDesc = Long
-internal const val VARINT = 0
-internal const val i64 = 1
-internal const val SIZE_DELIMITED = 2
-internal const val i32 = 5
+
+internal enum class ProtoWireType(val typeId: Int) {
```

Comment:
This could be a value class.

Reply:
Oooops, I know that it's sealed, not suitable for using value class. Nevermind for this.

Reply:
> it's sealed, not suitable for using value class.

You got the point